test/common/orca/orca_parser_fuzz.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package test.common.upstream;
+
+import "validate/validate.proto";
+import "envoy/extensions/load_balancing_policies/client_side_weighted_round_robin/v3/client_side_weighted_round_robin.proto";
+
+message OrcaParserTestCase {
+  string header_name = 1 [(validate.rules).string = {pattern: "([A-Za-z0-9-]+)$", max_bytes: 60000}];
+  string header_value = 2 [(validate.rules).string = {pattern: "([^\0^\r^\n]+)$"}];
```

Comment:
Will it make sense to replace this by a [`test.fuzz.Headers`](https://github.com/envoyproxy/envoy/blob/94f71389fae5dc3c2a7a846da58e90bd04706061/test/fuzz/common.proto#L15)?
If there are other restrictions for the name/value, I think the parser code should handle it (not the fuzzer).

Reply:
Great idea! Let me do that.

Reply:
Done, PTAL.